src/main/drivers/system.c

```diff
@@ -175,6 +175,12 @@ int32_t clockCyclesTo10thMicros(int32_t clockCycles)
     return 10 * clockCycles / (int32_t)usTicks;
 }
 
+// Note that this conversion is signed as this is used for periods rather than absolute timestamps
+int32_t clockCyclesTo100thMicros(int32_t clockCycles)
```

Comment:
Needs to be added to unit test

Reply:
alternatively you can add system.c to the makefile of the unit tests and system.h to the unit test